### ISSUE TITLE
Compatibility with Health Bar

### ISSUE THREAD

**Issue by filoghost** [jrbudda/Sentry#120]
Hello, I just want to ask you if you can make our plugins compatible. Look at this http://dev.bukkit.org/server-mods/health-bar/pages/compatibility/ or just use entity.setHealth() and entity.setMaxHealth(), or add an API please :)

**Comment by jrbudda** [comment]
Try with v1.3.6, available on the wiki.

**Comment by mcmonkey4eva** [comment]
Negative on 1.3.6 fixing it - their health shows wrong (Though I'm not using that plugin, I'm using the default minecraft health scoreboard option.) - All Citizens NPCs are 0 health, all Sentry NPCs are 20 health. (They sometimes go below 20 though.)

(also, @jrbudda - did you forget to put the 1.3.6 update into the github repo? Last posted change is 1.3.4, 18 days ago...)

**Comment by BriannaThorez** [comment]
I had this issue to and turned off the above or beside healthbar(the player one) that was graphic. In its place I enabled default custom health bar to true and used the Name - health/totalhealth preconfigured custom. Works like a charm on every thing. Then I set to hidden untill toggled so you dont know health untill you hit something. It immediately shows health on all. Seems to work for me now perfectly
